templates/default/custom.properties.erb

```diff
@@ -61,9 +77,9 @@ org.opencastproject.workspace.rootdir=${org.opencastproject.shared_storage.dir}/
 # The location to store uploaded static files such as images and videos.
 org.opencastproject.staticfiles.rootdir=${org.opencastproject.shared_storage.dir}/staticfiles
 
-# The ID of the default workflow definition to run when media are ingested
-org.opencastproject.workflow.default.definition=ng-schedule-and-upload
-
+# Location of the temporary directory to build zip archives.
+# Default: ${org.opencastproject.storage.dir}/tmp/zip
+org.opencastproject.workflow.handler.workflow.ZipWorkflowOperationHandler.tmpdir=<%= @shared_storage_root %>/zip-tmp
```

Comment:
How big are these zip archives and how frequently are they created? Just trying to ascertain if doing zip operations over NFS is the best choice. Are they big enough that the (faster) local instance storage isn't safe or practical?

Reply:
We are not generating zips anymore. I believe this is here to better match the one from upstream OC.

Reply:
@lbjay @rute-santos 's right. This is just moving the existing param in the file to match OC's ordering.